kura/org.eclipse.kura.web2/src/main/java/org/eclipse/kura/web/client/ui/network/TabWirelessUi.java

```diff
@@ -117,7 +117,7 @@
 
     private static final String REGEX_PASSWORD_WPA = "^[ -~]{8,63}$";
     private static final String REGEX_PASSWORD_WEP = "^(?:[\\x00-\\x7F]{5}|[\\x00-\\x7F]{13}|[a-fA-F0-9]{10}|[a-fA-F0-9]{26})$";
-    private static final int MAX_WIFI_CHANNEL = 13;
+    private static final int MAX_WIFI_CHANNEL = 14;
```

Comment:
I think you forgot to change the header in this file, it should be "Copyright (c) 2011, 2020 Eurotech and/or its affiliates"